examples/routing-with-resources/home.tsx

```diff
@@ -17,18 +17,23 @@ export const homeResource = createResource({
 export const Home = () => {
   // eslint-disable-next-line
   const { data, loading, error } = useResource(homeResource);
+  const breeds = data?.message ? Object.keys(data.message) : [];
 
   return (
     <div>
-      <h1>Home</h1>
-      <Link to={aboutRoute} prefetch="hover">
-        Go to about
-      </Link>
+      <h1>Dog Breeds</h1>
       <section>
-        <p>A random picture of a cute dog</p>
-        <section>
-          {!loading && <img src={data?.message} alt="A cute dog!" />}
-        </section>
+        <ul>  
+          {
```

Comment:
Nit: seems like prettier didn't run here (noticed the eslint command only checks for `/src` 😅)
Can you update `lint` command to include examples folder too?